Assets/Scripts/Features/UI/Input Windows/InputSetupWindow.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Setup windows can inherit from this class.
+/// Inheriting from this class requires your window to have a Text component for input that you can apply.
```

Comment:
When a non-static component is required, it would be useful to have the script use the [RequireComponent()] attribute, so that every script inheriting from this one will automatically add the required component when added to an object.
In this case it would be [RequireComponent(Typeof(Text))].